src/components/navbar/index.d.ts

```diff
@@ -0,0 +1,46 @@
+import { BulmaComponent } from '..';
+import { Color } from '..';
+
+interface NavbarProps {
+  transparent?: boolean;
+  fixed?: 'top' | 'bottom';
+  size: 'large';
```

Comment:
Is this supposed to be a mandatory prop? Seems especially odd since there's only one allowed value.

Reply:
you are right. this is an optional prop.